frontend/src/layouts/ContentLayout.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { CssBaseline, Paper } from '@mui/material';
+
+const styles = {
+  main: {
+    width: 'auto',
+    display: 'block', // Fix IE 11 issue.
+    marginLeft: 100,
+    marginRight: 100,
+  },
+  paper: {
+    marginTop: 3,
+    paddingBottom: 20,
+    flexDirection: 'column',
+    alignItems: 'center',
+    p: 2,
+    width: 900,
```

Comment:
Shouldn't hardcode a pixel amount for this, I think (?) as it would create problems in mobile/lower resolution systems.

Reply:
I tried to view the page with a mobile screen and I think 900 works like an upper bound instead of a lower bound:
- If the page width is higher than 900 pixels, content width remains 900 pixels.
- If the page is smaller than 900 pixels, then the content occupies the whole screen horizontally.
- If the page is smaller than 240 pixels, then things like what you show in your screenshot above occur.

I believe in the first two cases, the app is working as intended. I think we can neglect the case when the screen width is less than 240 pixels. What was the screen resolution you used when you generated the screenshot?

Reply:
The generated screenshot is in 390x844 (iphone resolution), so i don't think it is limited to 240p.
<img width="596" alt="resim" src="https://user-images.githubusercontent.com/26660856/202856917-e20e0904-613c-48ce-bd25-7f87f411a2eb.png">

 For case one i think `width: '100%', maxWidth: 900` or we can use `vh, vw` like `width: '95vh'` to set the maximum width. This screenshot is from a test with `maxWidth: '95vh'` instead of set width of 900. The overflow of date values can be easily solved as it can't be wrapped right now since there is no whitespace between characters. 
![resim](https://user-images.githubusercontent.com/26660856/202857206-0dede897-c622-439a-b970-099a945d5f80.png)

Also we could configure the grid for both smaller and medium+ size screens differently.  The image below is configured as:

```JSX
      <Grid container spacing={2}>
        <Grid item xs={12} sm={8}>
          <img src={artitem.artItemInfo.imageUrl} alt="Art Item" style={{width:'100%'}}/>
        </Grid>
        <Grid item xs={12} sm={4}>
```
setting the grid layout differently for screen sizes. (xs=xsmall < 600px, sm=small > 600px) Look at: https://mui.com/material-ui/react-grid/#grid-with-multiple-breakpoints & https://mui.com/material-ui/customization/breakpoints/#default-breakpoints

![resim](https://user-images.githubusercontent.com/26660856/202857563-6703d41e-35c2-4850-978b-30b4e7adccbe.png)

Reply:
Resolved in 411489d